src/libraries/Common/src/System/Net/Http/aspnetcore/Http2/Hpack/Huffman.cs

```diff
@@ -2,11 +2,14 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Diagnostics;
+using System.Threading;
 
 namespace System.Net.Http.HPack
 {
     internal class Huffman
     {
+        private static ushort[]? s_decodingTree;
+
         // TODO: this can be constructed from _decodingTable
```

Comment:
_decodingTable is being deleted

Reply:
Shall I just remove comment or add github link to former code?

Reply:
Is the comment still relevant to s_decodingTree, i.e. could/should the encoding table also be constructed dynamically?  If yes, you can update the comment (or address it).  If not, you can delete it.

Reply:
I think that _encodingTable shall not be generated. We need either _encodingTable or _decodingTable just to be able to generate  _encodingTable from it.
Both are about the same size:
![image](https://user-images.githubusercontent.com/25249058/96896303-a40a1300-148d-11eb-866b-45bc9446b3a7.png)
but _decodingTable has more heap allocations.
I will replace the comment by refernce to RFC from which is this table build

Reply:
done